test/fixtures/publications.yml

```diff
@@ -1,60 +1,61 @@
-# Read about fixtures at http://ar.rubyonrails.org/classes/Fixtures.html
```

Comment:
if these have now all be replaced with a Factory, may was well just remove all these rather than leaving commented out.